mern-ecommerce/controllers/user.js

```diff
@@ -125,3 +125,19 @@ exports.purchaseHistory = (req, res) => {
       res.json(orders);
     });
 };
+
+exports.deleteUser = (req, res) => {
```

Comment:
The `deleteUser` function is missing a JSON response object in the error handling block. It should be:

```javascript
return res.status(400).json({
  error: 'User not found',
});
```

Instead of just `return res.status(400).`.